vite.config.mjs

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react-swc'
+
+/// <reference types="vitest" />
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    include: ['__tests__/**/*.test.tsx'],
+    exclude: ['__tests__/fixtures/**'],
+    snapshotFormat: {
+      printBasicPrototype: true,
```

Comment:
Keep snapshots as close to the jest snapshots as possible https://vitest.dev/guide/snapshot#_2-printbasicprototype-is-default-to-false